modules/pvm-data/src/node_types.rs

```diff
@@ -86,12 +145,21 @@ impl fmt::Display for PVMDataType {
     }
 }
 
+impl PVMDataType {
+    pub fn compatable_concrete(self, ty: &ConcreteType) -> bool {
```

Comment:
should this be `compatible_concrete` instead?

Reply:
Yeah, that's a typo

Reply:
Corrected